### ISSUE TITLE
Past Challenges must be accessible by searching with the date filter

### ISSUE THREAD

**Issue by SathyaJayabal** [topcoder-platform/community-app#4972]
How do you access past challenges?

Exit:
Past Challenges must be accessible by searching with the date filter

**Comment by Oanh-and-only-Oanh** [comment]
@SathyaJayabal, Past challenges should be searched view filter with date rage

**Comment by LieutenantRoger** [comment]
@SathyaJayabal  Correct, the past challenges bucket is removed, please close this ticket. @Oanh-and-only-Oanh

**Comment by Oanh-and-only-Oanh** [comment]
@SathyaJayabal please verify that Past challenges should be searched view filter with date rage before closing. I was not able to do so... @LieutenantRoger

**Comment by SathyaJayabal** [comment]
@LieutenantRoger, but you are not able to search past challenges using keyword search nor the date filter
<img width="1440" alt="Screenshot 2020-09-21 at 10 12 15 PM" src="https://user-images.githubusercontent.com/58783823/93795854-9e839800-fc57-11ea-8431-df9d2a92c6e3.png">

**Comment by Oanh-and-only-Oanh** [comment]
Closing issue. 

It's not an issue - we allow members who submit in the last round to see the source of the others.
this was something we actually wanted - it has not always worked for design - at some point last year it went live/started working. Correct functionality is that if you submit to the challenge (final round of a challenge) a benefit is you can see the source of the other members and learn from it.

**Comment by LieutenantRoger** [comment]
@SathyaJayabal 

Please change the ticket description about what the problem, instead of posting a snapshot in the conversation. 

The main description apparently a false one.

**Comment by LieutenantRoger** [comment]
PR is here: https://github.com/topcoder-platform/community-app/pull/4993

**Comment by luizrrodrigues** [comment]
@LieutenantRoger I'm still seeing the issue.
http://local.topcoder.com/challenges?startDateStart=2018-07-01T15:00:00.000Z&endDateEnd=2018-10-31T15:00:00.000Z

![image](https://user-images.githubusercontent.com/7029641/94094503-08d23f00-fdf6-11ea-80ff-fdfaec9e1b6c.png)

**Comment by luizrrodrigues** [comment]
@SathyaJayabal @LieutenantRoger 

This looks like a issue with API, if check same range Prod also will return nothing:
https://www.topcoder.com/challenges?filter[startDate]=2018-07-01T15%3A00%3A00.000Z&filter[endDate]=2018-10-31T15%3A00%3A00.000Z&bucket=past

--------

@LieutenantRoger I see the changes and is a valid fix, because was missing `&` between params. So I'll merge the code.

--------

@LieutenantRoger But found a new issue, if refresh the page or try access directly (https://test-community-app.topcoder-dev.com/challenges?startDateStart=2020-08-01T15:00:00.000Z&endDateEnd=2020-08-31T15:00:00.000Z) filter not applied.

@SathyaJayabal Can you create a new issue to this one or maybe we can track here.

**Comment by SathyaJayabal** [comment]
@luizrrodrigues , I think #4981 will take care of the new issue. Please confirm.

**Comment by luizrrodrigues** [comment]
@SathyaJayabal Oh, yep.

**Comment by luizrrodrigues** [comment]
@SathyaJayabal @LieutenantRoger Now able to reproduce the issue:

Prod: https://www.topcoder.com/challenges?filter[startDate]=2018-07-01T15%3A00%3A00.000Z&filter[endDate]=2018-12-31T14%3A00%3A00.000Z&bucket=past

![image](https://user-images.githubusercontent.com/7029641/94096280-125da600-fdfa-11ea-821e-19e562bf9848.png)

Local:
http://local.topcoder.com/challenges?startDateStart=2018-07-01T15:00:00.000Z&endDateEnd=2018-12-31T14:00:00.000Z

![image](https://user-images.githubusercontent.com/7029641/94096358-4769f880-fdfa-11ea-8dab-98a0e265e821.png)

------------

Issue: FE always use `status=Active` in API call:
https://api.topcoder.com/v5/challenges/?startDateStart=2018-07-01T15%3A00%3A00.000Z&endDateEnd=2018-12-31T14%3A00%3A00.000Z&status=Active&isLightweight=true&perPage=1&tracks[]=Dev&tracks[]=Des&tracks[]=DS&tracks[]=QA

**Comment by SathyaJayabal** [comment]
@sushilshinde @luizrrodrigues , as discussed, we are waiting on clarification on how to implement this and where to display the past challenges

**Comment by ThomasKranitsas** [comment]
You can use the `endDateEnd` query parameter to fetch past challenges and set it to the current date. Additionally, you can also use `status=Completed` to only fetch the completed past challenges.

Example API call: http://api.topcoder.com/v5/challenges?endDateEnd=2020-09-24T00:00:00Z

**Comment by luizrrodrigues** [comment]
@LieutenantRoger Any status in this one?

Please check: https://github.com/topcoder-platform/community-app/issues/4972#issuecomment-698086308

**Comment by luizrrodrigues** [comment]
@Oanh-and-only-Oanh F2F https://www.topcoder.com/challenges/e661f904-9cf4-4812-ac71-1dc615d1e374

**Comment by LieutenantRoger** [comment]
Hi @luizrrodrigues  saw a challenge is launched for this issue. Yes, I used `status=Active` by default which causes this.

**Comment by SathyaJayabal** [comment]
@luizrrodrigues @sushilshinde @Oanh-and-only-Oanh , This looks good. 
<img width="1440" alt="Screenshot 2020-10-08 at 5 54 42 PM" src="https://user-images.githubusercontent.com/58783823/95458104-5ff81800-098f-11eb-8c84-594cd1cc2346.png">
